axon-server/administration/cluster-template.md

```diff
@@ -0,0 +1,84 @@
+#Cluster template
+
+With the Cluster Template feature we enabled users to maintain cluster configuration as a code.
+
+Cluster template is defined as a YAML file which describes cluster configuration. It is possible to pre-define replication groups, contexts, metadata, applications (with tokens), users and their roles and that configuration can be shared across teams.
+
+{% hint style="warn" %}
+Cluster template runs exactly once, on the first clean Axon Server start-up, if there is no previous cluster configuration defined. Therefore, cluster template WILL NOT override any existing configuration, its purpose is to be used during active development and to share cluster configuration across the developing team.
+{% endhint %}
+
+{% hint style="warn" %}
+As clustering is an Axon Server EE feature, license needs to be provided before using this feature.
+{% endhint %}
+
+
+## Usage
+
+To use Cluster template all you need to do is define a valid cluster template yml file. If this file is present on a clean Axon Server startup, it will automatically be picked up, and the cluster will be configured accordingly.
```

Comment:
```suggestion
To use the cluster template feature, all you need to do is define a valid cluster template yml file. If this file is present on a fresh Axon Server startup, it will automatically be picked up and the cluster will be configured accordingly.
```